website/docs/CBD.md

```diff
@@ -0,0 +1,106 @@
+---
+slug: cbd
+---
+
+# Condition Based Decryption
+
+Condition Based Decryption (CBD) is a programmable access control service, in which decryption rights are based on the verified fulfillment of predefined conditions.
+Access conditions can be EVM-based (e.g. does the designated recipient own this NFT), RPC-driven (e.g. did the designated recipient commence active mining before this block) or time-based (e.g. has a preset period of time elapsed, after which the recipient's requests will be ignored).
```

Comment:
```suggestion
Access conditions can be EVM-based (e.g. does the requester own this NFT?), RPC-driven (e.g. has the requester sent more than N transactions?) or time-based (e.g. has a preset period of time elapsed, after which the recipient's requests will be ignored).
```
The mining example is extremely confusing, because for starters, it'd be using the `eth_mining` RPC endpoint but evaluating it for each Ursula's ETH provider client, not for the requester. I propose using an account's transaction count, which is something that it's not accessible from the EVM but that can be queried with the RPC API. 

Also the notion of "designated recipient" it's a bit restrictive, and in a way is the same angle that PRE has. Instead, we should be talking about "requester", who can be anyone that fulfills the conditions.